x-pack/plugins/security/server/audit/audit_service.ts

```diff
@@ -61,3 +189,40 @@ export class AuditService {
     }
   }
 }
+
+export const createLoggingConfig = (config: ConfigType['audit']) =>
+  map<Pick<SecurityLicenseFeatures, 'allowAuditLogging'>, LoggerContextConfigInput>((features) => ({
+    appenders: {
+      auditTrailAppender: config.appender ?? {
+        kind: 'console',
+        layout: {
+          kind: 'pattern',
+          highlight: true,
+        },
+      },
+    },
+    loggers: [
+      {
+        context: 'audit.beta',
```

Comment:
nit: not a beta logger anymore

```suggestion
        context: 'audit',
```